src/codebook/data/summarize/continuous.js

```diff
@@ -9,7 +9,7 @@ export default function continuous(vector, sub) {
   const statistics = {};
   statistics.N = vector.length;
   const nonMissing = vector
-    .filter(d => !isNaN(+d.value) && !/^\s*$/.test(d.value))
+    .filter(d => d => !d.missing)
```

Comment:
why do I see two => here?

Reply:
That's a bug (create #306 to track). Good catch.